docs/source/specs/openapi.json

```
@@ -976,6 +976,20 @@
           },
           {
             "$ref": "#/components/parameters/OrderByFilter"
+          },
+          {
+            "name": "add_fields",
+            "in": "query",
+            "required": false,
+            "description": "Parameter for add list of fields to display for roles.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "string"
```

I think it might be good to add the values allowed here?
```suggestion
                "type": "string",
                "enum": [
                  "groups_in",
                  "groups_in_count"
                ]
```

---

Thanks, this is really good